README.md

```diff
@@ -170,11 +176,28 @@ After the above command completes, rerun the commands given at [Jupyter Classic
 
 ## Connecting to a graph database
 
+Configuration options can be set using the `%graph_notebook_config` magic command. The command accepts a JSON object as an argument. The JSON object can contain any of the configuration options listed below. The command can be run multiple times to change the configuration. The configuration is stored in the notebook's metadata and will be used for all subsequent queries.
+
+| Configuration Option | Description | Default Value | Type |
+| --- | --- | --- | --- |
+| auth_mode | the authentication mode to use for Amazon Neptune connections | DEFAULT | string |
```

Comment:
Capitalize the first letter in the description for each field.

Reply:
Will do